packages/stylelint-config-airbnb/index.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  plugins: [
+    'stylelint-scss',
+  ],
+  rules: {
+    'at-rule-empty-line-before': 'never',
```

Comment:
i'm kind of surprised that this plugin doesn't have a prefix on its rule names - ie, `stylelint-scss/indentation` - to avoid the risk of colliding with core eslint rules :-/